src/App.js

```diff
@@ -21,6 +22,7 @@ function App() {
         <Route path="/" element={<AppLayout />}>
           <Route path="/" element={<Home />} />
           <Route path="/watch/:watchid" element={<SingleVideoPage />} />
+          <Route path="results" element={<SearchResult />} />
```

Comment:
route can be written as `/results` to differentiate as subroute

Reply:
thanks for pointing, I'll correct it